src-tauri/src/main.rs

```diff
@@ -1,20 +1,12 @@
 #![cfg_attr(not(debug_assertions), windows_subsystem = "windows")]
 
+use std::collections::HashMap;
```

Comment:
what formatter are we you using? this would be hard for me to carefully review, can we only change code related to migration and keep the same formatting as before?

Reply:
otherwise everything else looks good to me, thank you 👍🏻

Reply:
It has been formatted using the default rust lang support plugin for vscode.

I have reverted my changes, please take a look at it again